App.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect } from 'react';
+
+const App = () => {
+    const [todos, setTodos] = useState([]);
+    const [newTodo, setNewTodo] = useState('');
+
+    useEffect(() => {
+        fetch('http://localhost:3000/todos')
```

Comment:
Error handling is not there 
if response.ok, then it execute rest ot the code 
try{}catch(error)
new Error ()